multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/madara/MadaraGenerator.kt

```diff
@@ -207,7 +207,7 @@ class MadaraGenerator : ThemeSourceGenerator {
         SingleLang("Manga Read", "https://mangaread.co", "en", overrideVersionCode = 1),
         SingleLang("Manga Rock Team", "https://mangarockteam.com", "en", overrideVersionCode = 1),
         SingleLang("Manga Rocky", "https://mangarocky.com", "en", overrideVersionCode = 1),
-        SingleLang("Manga Starz", "https://mangastarz.com", "ar", overrideVersionCode = 3),
+        SingleLang("Manga Starz", "https://mangalike.org/", "ar", overrideVersionCode = 4),
```

Comment:
remove trailing slash
```suggestion
        SingleLang("Manga Starz", "https://mangalike.org", "ar", overrideVersionCode = 4),
```